test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -572,5 +574,22 @@ TEST_F(QuicPlatformTest, FailToPickUnsedPort) {
   EXPECT_DEATH_LOG_TO_STDERR(QuicPickUnusedPortOrDie(), "Failed to pick a port for test.");
 }
 
+TEST_F(QuicPlatformTest, TestEnvoyQuicBufferAllocator) {
+  bool deterministic_stats = Envoy::Stats::TestUtil::hasDeterministicMallocStats();
+  const size_t start_mem =
+      deterministic_stats ? Envoy::Memory::Stats::totalCurrentlyAllocated() : 0;
```

Comment:
actually you don't need to conditionalize this; totalCurrentlyAllocated() will return 0 if deterministic_stats is false.

Reply:
done